src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -1,4 +1,7 @@
 <div class="view-content columns">
+    <span *ngIf="pluginInfo?.type === 'South' || pluginInfo?.type === 'North' || pluginInfo?.type === 'Filter'" class="icon is-small tooltip is-pulled-right is-hovered" data-tooltip="Help" (click)="navToLink()">
```

Comment:
better to use includes?

Reply:
pluginInfo is an object.

Reply:
I mean ['South', 'North', 'Filter'].includes(pluginInfo?.type);